superset-frontend/src/cccs-viz/plugins/plugin-chart-ag-grid/src/ag-grid/AGGridViz.tsx

```diff
@@ -448,6 +450,28 @@ export default function AGGridViz({
           }
           key="submit-file-to-assembly-line"
           data={selectedData.typeData.eml_path}
+          base_url={assemblyLineUrl}
+          disabled={selectedData.typeData.eml_path.length > SUBMISSION_LIMIT}
+          tooltip={
+            new Set(selectedData.typeData.eml_path).size > SUBMISSION_LIMIT
+              ? `A window will open for each row selected. You cannot submit more than ${SUBMISSION_LIMIT} eml files at a time.`
+              : undefined
+          }
+        />,
+      ];
+    }
+    if (
+      selectedData.typeData.eml_path &&
+      selectedData.typeData.eml_path.length > 0
+    ) {
+      specialMenuItems = [
+        ...specialMenuItems,
+        <DownloadEmailMenuItem
```

Comment:
Is the diff just being weird, or has FLOW-8 snuck in here?

Reply:
If flow-8 was merged into 3.1, and you've since merged 3.1 in here, it's a bit suspicious that this is showing up in the diff.

Reply:
Code was checked, seems correct

Reply:
Whatever you've done, this seems to have vanished from the diff now, so should be good! :)